### PR TITLE
More Loadout Equipment

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This PR adds a bunch more equipment selections for Engineering loadouts, plus Suit/Skirt selections for the jobs that were missing it.

<details><summary><h1>Media</h1></summary>
<p>

image

</p>
</details>

# Changelog

:cl:
- add: All engineering roles have had their equipment loadouts significantly expanded upon. Engineers can now buy construction materials with their loadout points. 
- fix: All engineering jobs now have their Suit/Skirt selection via loadouts.
- add: Salvage techs can now select from a variety of knife options to start their spess adventures with.
- add: Epistemics staff now have *some* equipment selection options that they share. More to come when I finish making the Potentiometer.

### CODE REVIEW

**Comment by VMSolidus** [comment]
> I am fully against this PR as this just bloat loadout, loadout are to customize your character looks not fully replace equipements, ect...
> 
> Lockers exist for a reason and getting materials in some parts, this just nullifies it.

image

**Comment by rbertoche** [comment]
TLDR; Basic tools to get your job done would be better as an opt-out, not an opt-in.

**Comment by VMSolidus** [comment]
I have given salvage players their own special attention. You're welcome.

image

**Comment by VMSolidus** [comment]
Test fails are *more or less* unrelated to this PR. 
Y'all clowns need to review this before I add 300 more items to it.

**Review by sleepyyapril — APPROVED** [review]
Considering these is YML and FTL changes and there's no linter screaming, this is fine.

**Review by Peptide90 — APPROVED** [review]

**Review by Remuchi — APPROVED** [review]
fine I guess?
